kuma/users/stripe_utils.py

```diff
@@ -99,10 +99,13 @@ def cancel_stripe_customer_subscriptions(user):
     """Delete all subscriptions for a Stripe customer."""
     assert user.stripe_customer_id
     customer = stripe.Customer.retrieve(user.stripe_customer_id)
+    canceled = []
     for sub in customer.subscriptions.data:
         s = stripe.Subscription.retrieve(sub.id)
         UserSubscription.set_canceled(user, s.id)
         s.delete()
+        canceled.append(s)
+    return canceled
```

Comment:
smuggled a fix for an unrelated problem in, huh? 😏 I'm all for fixes

Reply:
I have no memory of that! Gah!! I wonder if I accidentally worked on the wrong branch or something.

Reply:
Oh I guess it would've been more prudent for me to postpone merging after you replied whether it was intentional. Will do so next time!